src/ReleaseHistory.md

```diff
@@ -358,3 +358,8 @@
 * API BREAKING: `threadFlowLocation.executionOrder` can be 0 and defaults to -1, so minimum: -1, default: -1
 * API BREAKING: Rename artifact role `traceFile` to `tracedFile`.
 * API NON-BREAKING: Add artifact role `debugOutputFile`.
+* API NON-BREAKING: Add a new value to `result.kind`: `informational`.
+* API NON-BREAKING: add `address.kind`�values `function`�and `page`.
+* API NON-BREAKING: `run.columnKind`�has no default value.
```

Comment:
>� [](start = 36, length = 1)

do you know what these unicode characters are in the doc? a strange non-breaking space or something? try opening in notepad and stripping them.

Reply:
then merge!

---
In reply to: [278141596](https://github.com/Microsoft/sarif-sdk/pull/1455#discussion_r278141596) [](ancestors = 278141596)

Reply:
woah! this looks ugly! let me remove it :)

---
In reply to: [278141687](https://github.com/Microsoft/sarif-sdk/pull/1455#discussion_r278141687) [](ancestors = 278141687,278141596)